RecoPixelVertexing/PixelTriplets/plugins/CAHitQuadrupletGenerator.h

```diff
@@ -38,8 +37,8 @@ class CAHitQuadrupletGenerator : public HitQuadrupletGenerator {
 
 public:
 
-    CAHitQuadrupletGenerator(const edm::ParameterSet& cfg, edm::ConsumesCollector&& iC, bool needSeedingLayerSetsHits=true): CAHitQuadrupletGenerator(cfg, iC, needSeedingLayerSetsHits) {}
-    CAHitQuadrupletGenerator(const edm::ParameterSet& cfg, edm::ConsumesCollector& iC, bool needSeedingLayerSetsHits=true);
+    CAHitQuadrupletGenerator(const edm::ParameterSet& cfg, edm::ConsumesCollector&& iC): CAHitQuadrupletGenerator(cfg, iC) {}
+    CAHitQuadrupletGenerator(const edm::ParameterSet& cfg, edm::ConsumesCollector& iC);
 
     virtual ~CAHitQuadrupletGenerator();
```

Comment:
I suppose is in https://cmssdt.cern.ch/SDT/code-checks/PR-20177/133/git-diff.patch
that in my understanding has not been applied yet...